packages/flutter/lib/src/gestures/drag.dart

```diff
@@ -63,10 +63,22 @@ bool _isFlingGesture(Velocity velocity) {
 }
 
 abstract class _DragGestureRecognizer<T extends dynamic> extends OneSequenceGestureRecognizer {
+  /// A pointer has contacted the screen and might begin to move.
   GestureDragDownCallback onDown;
+
+  /// A pointer has contacted the screen and has begun to move.
   GestureDragStartCallback onStart;
+
+  /// A pointer that is in contact with the screen and moving has moved again.
   _GesturePolymorphicUpdateCallback<T> onUpdate;
```

Comment:
I just noticed this is a public property whose value is a private type. How can clients actually use this property if the type is private?